lib/helper/Protractor.js

```diff
@@ -224,6 +224,22 @@ class Protractor extends SeleniumWebdriver {
   }
 
   /**
+   * {{> ../webapi/waitForInvisible }}
+   */
+  waitForInvisible(locator, sec) {
+    sec = sec || 1;
+    let el = this.browser.element(guessLocator(locator) || by.css(locator));
+    return this.browser.wait(EC.invisibilityOf(el), sec*1000);
+  }
+
+  /**
+   * {{> ../webapi/waitForNotAttached }}
+   */
+  waitForNotAttached(locator, sec) {
+    return this.waitForInvisible(locator, sec);
```

Comment:
There isn't, in Protractor `invisibilityOf` does hidden or not attached to the DOM. We can have `stalenessOf` which does not attached to the DOM but that would give the same result at the end.